scripts/e2e.ts

```diff
@@ -29,6 +29,21 @@ function publishPackage(packagePath) {
   }
   try {
     console.log(` 📦 ${packagePath}`);
+
+    // NPM@7 requires a token to publish, thus, is just a matter of fake a token to bypass npm.
+    // See: https://twitter.com/verdaccio_npm/status/1357798427283910660
+    if (npmMajorVersion === 7) {
```

Comment:
Does this break for `npm@6`? Also, if we need to check then we should do greater or equals so that it doesn't break later on.

Reply:
I think it doesn't break for `npm@6`. I just wanted to be explicit for this, as a workaround :) and I'm pretty confident that it another workaround will be needed when `npm@8` is released :D